docs/reference/search/search.asciidoc

```diff
@@ -552,8 +553,8 @@ Period of time used to extend the life of the PIT.
 
 [[request-body-rank]]
 `rank`::
-(Optional, object) Defines a method for combining and ranking `1` standard query
-with `1..n` knn searches or no standard query with `2..n` knn searches.
+(Optional, object) Defines a method for combining and ranking result sets from
```

Comment:
would it make sense to mention the RRF ranker and link to it?

Reply:
RRF is included as part of the ranking methods right below this.